assets/quick-add-bulk.js

```diff
@@ -3,6 +3,7 @@ class QuickAddBulk extends HTMLElement {
   constructor() {
     super();
     const debouncedOnChange = debounce((event) => {
+      this.cleanErrors();
```

Comment:
Let's remove these changes since this is part of this PR

https://github.com/Shopify/dawn/pull/3321

```suggestion
```

Reply:
Actually it is not. I just used the same approach to clean errors.

Reply:
Ah 👀 I see. This is a great thing to extract it in the future so we can re-use it for both elements since the functions are so similar

We can add it as a comment here https://github.com/Shopify/dawn/issues/3247